immune_subtype_clustering/nf_launch.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import sys
+
+from orca.services.nextflowtower import NextflowTowerOps
+from orca.services.nextflowtower.models import LaunchInfo
+
+from utils import monitor_run, configure_logging
+
+
+def nextflow_execute_workflow(run_name: str, s3_file: str, cwl_file: str):
```

Comment:
For future rework, there is duplicate code for launching of workflows, but the `LaunchInfo` is what changes.  perhaps this calls for a `def nextflow_execute` function that takes in a `launchInfo` parameter.

Reply:
I was thinking the same, but did not think it was worth it for this particular task.